src/rules/orderedImportsRule.ts

```diff
@@ -51,16 +60,23 @@ export class Rule extends Lint.Rules.AbstractRule {
                     type: "string",
                     enum: ["case-insensitive", "lowercase-first", "lowercase-last"],
                 },
+                "source-imports-order": {
+                    type: "string",
+                    enum: ["case-insensitive", "lowercase-first", "lowercase-last"],
+                },
             },
             additionalProperties: false,
         },
-        optionExamples: ["true", '[true, {"named-imports-order": "lowercase-first"}]'],
+        optionExamples: [
+            "true",
+            '[true, {"named-imports-order": "lowercase-first", "source-imports-order": "lowercase-last"}]',
```

Comment:
My rationale was:

1. Breaking change if options are consolidated (renamed).
2. The initial implementation is actually 'lowercase-last' for source-imports and 'case-insensitive' for named-imports, which seems to indicate a use-case envisioned, at least, by the author.

I don't mind changing if you guys think that we should only have one option though.